feature-libs/cart/main/components/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.ts

```diff
@@ -83,11 +83,12 @@ export class AddedToCartDialogComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.addedEntryWasMerged$ = this.loaded$.pipe(
-      filter((loaded) => loaded),
-      switchMapTo(this.cartService.getEntries()),
-      map((entries) => entries.length === this.numberOfEntriesBeforeAdd)
-    );
+    this.addedEntryWasMerged =
+      getLastValueSync(
+        this.entry$.pipe(
+          map((cartEntry) => (cartEntry?.quantity ?? 0) > this.quantity)
```

Comment:
if `cartEntry.quantity` is 1, and adding 1 same product, the value will be `false`. But, it should be `true`.
![Screen Shot 2022-01-06 at 3 17 53 PM](https://user-images.githubusercontent.com/44440575/148441112-541536ca-f4ca-4e9f-a5e9-d8906ab1c15c.png)

Before, we checked `numberOfEntriesBeforeAdd`. If there are 8 entries, after adding new products, there are still 8 products, that means we don't add new products, but instead updating the existing quantity.

Reply:
I can repro.  This needs fixing.

Reply:
This is fixed.